Source/Core/VideoCommon/RenderState.cpp

```diff
@@ -65,6 +67,8 @@ void BlendingState::Generate(const BPMemory& bp)
   colorupdate = bp.blendmode.colorupdate && alpha_test_may_success;
   alphaupdate = bp.blendmode.alphaupdate && target_has_alpha && alpha_test_may_success;
   dstalpha = bp.dstalpha.enable && alphaupdate;
+  usedualsrc = g_ActiveConfig.backend_info.bSupportsDualSourceBlend &&
```

Comment:
The idea was to not include any backend/hardware states into this objects. So they will match on different computers/GPUs/backends.